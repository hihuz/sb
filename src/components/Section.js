import React from "react";

const Section = ({ name, title, text }) => (
  <section className="sb-section">
    <article className={name}>
      <h1>{title}</h1>
      {text.map(para => <p key={para}>{para}</p>)}
    </article>
  </section>
);

export default Section;
