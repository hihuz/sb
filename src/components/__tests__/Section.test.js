import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import Section from "../Section";

describe("Section", () => {
  test("Snapshot", () => {
    const component = shallow(<Section text={[]} />);
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
