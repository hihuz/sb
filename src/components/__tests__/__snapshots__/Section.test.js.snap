// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Snapshot 1`] = `
<section
  className="sb-section undefined"
>
  <article>
    <h1 />
    <div
      className="logo-container"
    >
      <img
        src="undefined.svg"
      />
    </div>
  </article>
</section>
`;
